app/controllers/admin/action_pages_controller.rb

```diff
@@ -66,6 +76,10 @@ def update
     @actionPage.og_image         = nil if params[:destroy_og_image]
 
     @actionPage.update_attributes(action_page_params)
+    if institutions_params[:reset] && institutions_params[:reset] == "1"
+      ActionInstitution.add(action_page: @actionPage,
```

Comment:
This should also trigger when `institutions_params[:category]` is present, otherwise the branch looks ready!

Reply:
done!

Reply:
The situation I was thinking of here is when affiliations are enabled for the first time on an already-existing petition. In that case, the category param is present but not the reset param, so the `ActionInstitution.add` call here is still missed and the page hits an error when it reloads, on 

```ruby
@institution_category = @institutions.first.category
```

Reply:
OH! that makes sense, just changed it. It only triggers when there's no institutions on the action.